src/file_organizer/updater/trust.py

```
@@ -0,0 +1,109 @@
+        canonical_bytes = json.dumps(manifest_data, sort_keys=True, separators=(",", ":")).encode(
+            "utf-8"
+        )
+
+        # Load the public key
+        pub_key = ECC.import_key(PINNED_PUBLIC_KEY)
+
+        # Decode signature content
+        sig_str = signature_content.strip()
+        try:
+            sig_bytes = base64.b64decode(sig_str)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)